packages/scxa-marker-gene-heatmap/src/MarkerGeneHeatmap.js

```diff
@@ -26,12 +26,20 @@ async function addModules () {
 addModules()
 
 const splitPhrase = (phrase, maxLineLength = 12) => {
-  const splitPhrase = [``]
+  const splitPhrase = []
 
   const words = phrase.split(/\s+/)
+
+  // Add first word to the first line if splitPhrase is empty
+  if (words.length > 0) {
+    splitPhrase.push(words.shift())
+  }
+
   while (words.length > 0) {
     const nextWord = words.shift()
-    if (splitPhrase[splitPhrase.length - 1].length + nextWord.length < maxLineLength) {
+
+    // Check if adding the next word would exceed the maxLineLength
```

Comment:
Could you please remove the comment and extract the if condition to a well named method?
That way we don't need the comment and the code is more readable.
Thanks